src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/DesignTimeTargets/Microsoft.Managed.DesignTime.targets

```diff
@@ -396,6 +396,12 @@
           DependsOnTargets="GetSuggestedWorkloads"
           Condition="'@(MissingWorkloadPack)' != '' And '$(UsingMicrosoftNETSdk)' == 'true'" />
 
+  <!-- This target is used to collect the MissingSdkRuntime items in the project.-->
+  <Target Name="CollectMissingSdkRuntimes"
```

Comment:
This target is unused and can be removed.

Reply:
done. Removed